x-pack/plugins/timelines/common/api/search_strategy/model/runtime_mappings.ts

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { z } from 'zod';
+
+export type MappingRuntimeFieldType =
+  | 'boolean'
+  | 'date'
+  | 'double'
+  | 'geo_point'
+  | 'ip'
+  | 'keyword'
+  | 'long'
+  | 'lookup';
+
+export const runtimeMappings = z
```

Comment:
Want to use this type for the `runtimeMappings` todo?

Reply:
yes, applied